annotation-file-utilities/src/main/java/org/checkerframework/afu/scenelib/io/IndexFileParser.java

```diff
@@ -486,6 +486,16 @@ private Object parseScalarAFV(ScalarAFT aft) throws IOException, ParseException
           if (type == long.class) {
```

Comment:
There is a `if (type == double.class)` about 8 lines above (and one for `float`).  Why not fold this logic into the existing tests?  The same goes for the `type == long.class` test on this line, for that matter.  (But I didn't notice it while reviewing that code.)